charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml

```diff
@@ -65,6 +65,23 @@ spec:
                           TODO: Add other useful fields. apiVersion, kind, uid?'
                         type: string
                     type: object
+                  image:
+                    type: string
+                  imagePullPolicy:
+                    description: PullPolicy describes a policy for if/when to pull
+                      a container image
+                    type: string
+                  imagePullSecrets:
+                    items:
+                      description: LocalObjectReference contains enough information
+                        to let you locate the referenced object inside the same namespace.
+                      properties:
+                        name:
+                          description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                            TODO: Add other useful fields. apiVersion, kind, uid?'
+                          type: string
+                      type: object
+                    type: array
```

Comment:
When I did the docket-build, as expect the opensearch-operator/config/crd/bases/opensearch.opster.io_opensearchclusters.yaml file was updated.

My question is even before my PR, the updated opensearch.opster.io_opensearchclusters.yaml content is different from charts/opensearch-operator/templates/opensearchclusters.opensearch.opster.io-crd.yaml 

**I am not sure this snippet is needed for PR or not. I can remove it if it is not needed.**